### PR TITLE
Remove more equivalents of the now method from the Painless whitelist.

### PR DESCRIPTION
**Pull request by jdconrad**
As the title says.

### CODE REVIEW

**Review by rjernst — APPROVED** [review]
Lgtm

File: modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.time.txt [hunk 1]
```diff
@@ -29,14 +29,6 @@ class Clock -> java.time.Clock extends Object {
   ZoneId getZone()
   Instant instant()
   long millis()
-  Clock offset(Clock,Duration)
```
Comment (nik9000) [inline modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.time.txt]: I *think* this one is ok?

File: modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.time.txt [hunk 2]
```diff
@@ -29,14 +29,6 @@ class Clock -> java.time.Clock extends Object {
...
-  Clock system(ZoneId)
-  Clock systemDefaultZone()
-  Clock systemUTC()
-  Clock tick(Clock,Duration)
```
Comment (nik9000) [inline modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.time.txt]: I think this one is OK too.

**Review by nik9000 — CHANGES_REQUESTED** [review]

File: modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.time.txt (hunk 1 above)
Comment (jdconrad) [inline modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.time.txt]: Added back.

File: modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.time.txt (hunk 2 above)
Comment (jdconrad) [inline modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.time.txt]: Added back.

**Comment by jdconrad** [comment]
@nik9000 Good to go here?

**Review by nik9000 — APPROVED** [review]

**Comment by nik9000** [comment]
I'm good now, yeah.

**Comment by jdconrad** [comment]
@rjernst @nik9000 Thanks for the reviews!

**Pull request merged**